package.json

```diff
@@ -1,73 +1,72 @@
 {
   "name": "app",
-  "private": true,
-  "description": "{{= appDescription }}'s website",
+  "description": "{{= appDescription }}",
   "version": "0.1.0",
   "author": "{{= appAuthor }}",
-  "keywords": [
-    "gatsby"
-  ],
-  "license": "MIT",
   "scripts": {
     "build": "gatsby build",
     "develop": "gatsby develop -H 0.0.0.0",
     "dev": "npm run develop",
     "serve": "gatsby serve",
     "start": "npm run develop",
-    "format": "prettier --write \"src/**/*.tsx\"",
-    "lint": "tslint -c tslint.json 'src/**/*.{ts,tsx}'",
-    "test": "concurrently \"npm run test:unit && npm run test:ts\"",
+    "format": "prettier --write src/**/*.{ts,tsx,js,jsx,md,json,yml,svg}",
+    "lint": "eslint src --ext .js,.jsx,.ts,.tsx",
+    "test": "npm run test:unit && npm run test:ts",
     "test:unit": "jest",
-    "test:ts": "tsc --skipLibCheck"
+    "test:ts": "tsc"
   },
   "dependencies": {
-    "@types/jest": "^24.0.13",
     "gatsby": "^2.6.0",
     "gatsby-image": "^2.1.0",
-    "gatsby-plugin-alias-imports": "^1.0.5",
-    "gatsby-plugin-favicon": "^3.1.6",
-    "gatsby-plugin-manifest": "^2.1.1",
-    "gatsby-plugin-netlify-cache": "^1.2.0",
-    "gatsby-plugin-react-helmet": "^3.0.12",
-    "gatsby-plugin-react-svg": "^2.1.1",
-    "gatsby-plugin-sharp": "^2.0.37",
-    "gatsby-plugin-sitemap": "^2.1.0",
-    "gatsby-plugin-typescript": "^2.0.14",
-    "gatsby-plugin-webpack-size": "^0.0.3",
-    "gatsby-source-filesystem": "^2.0.37",
-    "gatsby-transformer-sharp": "^2.1.19",
     "react": "^16.8.6",
     "react-dom": "^16.8.6",
     "react-helmet": "^5.2.1",
     "styled-components": "^4.2.0",
-    "wipe.css": "^1.2.0"
+    "wipe.css": "^1.1.1"
   },
   "devDependencies": {
+    "@babel/code-frame": "^7.0.0",
```

Comment:
do we need this one?

Reply:
Test needs it

Reply:
strange. I have jest in Significa.co and didn't need code-frame

Reply:
You're right, it is not required. I was just trying to solve a mistake in alias things